conversation_reconstruction/dataflow/write_page_states_to_spanner/dataflow_main.py

```diff
@@ -18,11 +18,17 @@
 
 Run with:
 
-  python dataflow_main.py --input_storage=YourInputDataInCloudStorage --spanner_instance=SpannerInstanceID
+  python dataflow_main.py
+         --input_storage=YourInputDataInCloudStorage
+         (Or) --bigquery_table=YourBigQueryTable
```

Comment:
It's a bit confusing to have (or) and (optional) in the example command line. Better to just show an example, and then note that instead of the "--input_storage" flag, you can provide the "--bigquery_table=" flag. 

And also write a little about what "--testmode" means.